app/v1/models/hateoas/RelType.scala

```diff
@@ -19,5 +19,9 @@ package v1.models.hateoas
 object RelType {
   val TRIGGER = "trigger"
 
+  val METADATA = "metadata"
+
+  val TAXABLEINCOME = "taxable-income"
```

Comment:
```suggestion
  val TAXABLE_INCOME = "taxable-income"
```

Change for readabillity.